tests/api/public_routes/test_heartbeat.py

```diff
@@ -0,0 +1,13 @@
+"""Test nestor_api.api.public_routes.heartbeat."""
+from flask.testing import FlaskClient
```

Comment:
Is this import necessary ?

Reply:
Changed so that pylint also checks on the test folder